c1688285.lua

```diff
@@ -25,19 +25,19 @@ function s.initial_effect(c)
 	e2:SetOperation(s.thop)
 	c:RegisterEffect(e2)
 	--material
+	local g=Group.CreateGroup()
+	local custom_code=aux.RegisterMergedDelayedEvent_ToSingelCard(c,id,EVENT_TO_GRAVE,g)
+	aux.RegisterMergedDelayedEvent_ToSingelCard(c,id,EVENT_REMOVE,g,custom_code)
```

Comment:
Is it a spelling mistake? This is OCG card and please avoid them

Reply:
changed